ci-operator/step-registry/openshift/e2e/gcp/libvirt/cert-rotation/conf/openshift-e2e-gcp-libvirt-cert-rotation-conf-commands.sh

```diff
@@ -0,0 +1,442 @@
+
+oc adm release mirror -a ~/pull-secret \
+--from="${RELEASE_IMAGE_LATEST}" \
+--to-release-image="${LOCAL_REG}/${LOCAL_REPO}:${OCP_RELEASE}" \
+--to="${LOCAL_REG}/${LOCAL_REPO}"
+
+# in case it's set in CI job yaml
+unset OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+
+# extract libvirt installer from release image
+oc adm release extract -a ~/pull-secret --command openshift-baremetal-install "${LOCAL_REG}/${LOCAL_REPO}:${OCP_RELEASE}"
+sudo mv openshift-baremetal-install /usr/local/bin/openshift-install
+
+# extract oc from release image
+oc adm release extract -a ~/pull-secret --command oc "${LOCAL_REG}/${LOCAL_REPO}:${OCP_RELEASE}"
+sudo mv oc /usr/local/bin/oc
+
+cat > "${CLUSTER_DIR}/install-config.yaml" << EOF
+apiVersion: v1
+baseDomain: "${BASE_DOMAIN}"
+compute:
+- hyperthreading: Enabled
+  architecture: amd64
+  name: worker
+  platform: {}
+  replicas: 2
+controlPlane:
+  hyperthreading: Enabled
+  architecture: amd64
+  name: master
+  platform: {}
+  replicas: 3
+metadata:
+  creationTimestamp: null
+  name: installer
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 192.168.126.0/24
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  libvirt:
+    network:
+      if: tt0
+publish: External
+pullSecret: $(echo \'"${PULL_SECRET}"\')
+sshKey: |
+  $(cat "${PUB_SSH_KEY}")
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  $(echo "${CA}")
+  -----END CERTIFICATE-----
+imageContentSources:
+- mirrors:
+  - ${HOSTNAME}:5000/ocp4/openshift4
+  source: quay.io/openshift-release-dev/ocp-release-nightly
+- mirrors:
+  - ${HOSTNAME}:5000/ocp4/openshift4
+  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+EOF
+
+# Create manifests and modify route domain
+openshift-install --dir="${CLUSTER_DIR}" create manifests
+# Workaround for https://github.com/openshift/installer/issues/1007
+# Add custom domain to cluster-ingress
+yq write --inplace $CLUSTER_DIR/manifests/cluster-ingress-02-config.yml spec[domain] apps.$BASE_DOMAIN
+
+# Add master memory to 14 GB
+# This is only valid for openshift 4.3 onwards
+yq write --inplace ${CLUSTER_DIR}/openshift/99_openshift-cluster-api_master-machines-0.yaml spec.providerSpec.value[domainMemory] 14336
+set -x
+openshift-install create cluster --log-level=debug --dir="$CLUSTER_DIR" || true
+openshift-install wait-for install-complete --log-level=debug --dir="$CLUSTER_DIR"
+END
+chmod +x "${SHARED_DIR}"/create-cluster-mirrored-local-registry
+
+cat  > "${SHARED_DIR}"/time-skew-test.sh << 'EOF'
+#!/bin/bash
+
+set -euxo pipefail
+
+approveCSRs () {
+  pendingCSRs=$(oc get csr | grep Pending | wc -l)
+  if [ $pendingCSRs -ne 0 ]; then
+    echo "Approving pending csrs"
+    oc get csr -o name | xargs oc adm certificate approve
+    sleep 30
+  fi
+}
+
+# TODO: Need to improve this nodesReady check
+checkNodesReady () {
+  nodesReady=0
+  retries=0
+  while [ $nodesReady -ne 5 ] && [ $retries -lt 30 ]; do
```

Comment:
you should probably read the number on start, looks like there is just one worker now https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/12878/rehearse-12878-periodic-ci-openshift-release-master-ocp-4.7-e2e-gcp-libvirt-cert-rotation/1352747737774297088#1:build-log.txt%3A68

Reply:
that was a flake, for sure, I haven't seen another run only get 1 worker.. i'll keep an eye out for it, once this merges, I'll take ownership to watch flakes